Stacks_Using_Linked_Lists/Stacks.c

```diff
@@ -0,0 +1,96 @@
+/* C program to implement stack using linked list*/
+
+#include<stdio.h>
+struct stack
+{
+int top;
+int data[100];
+}Stack; 
+
+/*Function to push element to stack */
+void PUSH(struct stack *s,int value,int n)
+{
+    if(s->top == n-1)
+        {
```

Comment:
Please fix indentation in all if-else blocks.

The block from line 69 to 78 looks good. Please follow that style everywhere.